docs/limitations.rst

```diff
@@ -29,6 +29,14 @@ some less obvious limitations when using some I/O libraries with UnifyFS
     corruption. Running the application through :doc:`VerifyIO <verifyio>` can
     help determine if any file locking calls are made.
 
+.. rubric:: Directories
+
+UnifyFS does not support most directory operations. Although directories can be
```

Comment:
May want to hedge this a bit since we are working toward better support for directories. How about "does not currently support"

Reply:
Cut it to the one line.